frontend/src/v5/ui/routes/viewer/drawings/drawingsList/drawingsList.styles.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ *  Copyright (C) 2024 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import styled from 'styled-components';
+import { Drawing } from './drawingItem/drawingtem.styles';
+
+export const List = styled.div`
```

Comment:
Is there a way to extrapolate this or the one in tickets to make it reusable and use it in both?
That way we can start building the v5 viewer components library.